torch/onnx/symbolic_opset9.py

```diff
@@ -2159,6 +2159,29 @@ def pixel_shuffle(g, self, upscale_factor):
                                     allowzero=0)
 
 
+@parse_args("v", "i")
+def pixel_unshuffle(g, self, downscale_factor):
+    dims = sym_help._get_tensor_sizes(self)
+    if len(dims) != 4:
+        return _unimplemented("pixel_shuffle", "only support 4d input")
+    if any([i is None for i in dims[1:]]):
+        return _unimplemented("pixel_shuffle", "only support static input shape, except for batch size")
```

Comment:
had an idea that possible can work for all dynamic input shape. maybe reshape 1 dim at a time with 0s to divide the height/weight, like `first_view = reshape(input, [0, 0, -1, scale, 0])`, then `second_view = reshape(input, [0, 0, 0, 0, -1, scale])`.

Reply:
Is there any way we could force the division to result in being an integer?
From what I understand, seems like [0, 0, -1, scale , 0] is essentially [0, 0, dim/scale, scale, 0] however we want the division to be 
```dim // scale```, otherwise we would get a floating point exception